### DISCUSSION TITLE
Add option to include link text

### DISCUSSION THREAD

**Discussion by flcdrg** [mjisaak/skilling-champion-extension#15] [Ideas]
When I'm posting a link to Twitter, I often want the link text as well as the URL. eg.

![image](https://user-images.githubusercontent.com/384747/111087395-4deb5880-8571-11eb-9f2a-7b472d9eb3b2.png)

I've prototyped one possible implementation for this that adds a second set of menu items. But maybe there are other ways this could be done too.

![image](https://user-images.githubusercontent.com/384747/111087784-6bb9bd00-8573-11eb-85ba-b09d5628e8f2.png)

I note that there appears to be an encoding issue (in Firefox at least) with the ampersand in the parent menu.

**Comment by mjisaak** [comment]
@flcdrg 
Hi David, 

this is a great idea. We could think about whether we want to add a second sub-menu for this (like in your screenshot) or we add a flag to the option like:

- [ ]  Copy links with link text

I converted the issue to an idea.
Best regards,
Martin

**Comment by flcdrg** [comment]
Nice.. this is my first 'idea' on GitHub 😀

Yeah.. I wondered about that too.

Depends how much you'd want to have both options available, or if you'd mostly just want it one way.
